src/plugins/dataflow/nodes/utilities/view-utilities.ts

```diff
@@ -60,10 +60,10 @@ export function moveNodeToFront(editor: NodeEditor, node: Node, newNode: boolean
 }
 
 export function hasFlowIn(node: Node){
-  const inputs = Array.from(node.inputs.values());
+  const inputs = Array.from(node.inputs.values()) as Input[];
   if (inputs.length === 0) return false;
   if (node.name === "Control") return inputs[0].connections.length > 0;
-  return inputs.some((input) => input.connections.length > 0);
+  return inputs.some((input: Input) => input.connections.length > 0);
```

Comment:
FYI, you probably don't need this change, since the type should propagate from the change above.

Reply:
Thanks, removed.